components/PoisList/CategorySelector.vue

```diff
@@ -65,17 +68,28 @@ export default defineNuxtComponent({
         .map((menuItem) => {
           const parents: string[] = []
           let parentId = menuItem.parent_id
+          let isIncluded = false
+
           while (parentId) {
             if (!this.menuItems[parentId])
               return undefined
 
+            if (
+              this.filters
+                && (this.filters.includes(parentId.toString()) || this.filters.includes(menuItem.id.toString()))
```

Comment:
```suggestion
                && this.filters.includes(menuItem.id.toString())
```
`parentId` will get checked on next loop.

Not better to turn `filters` content to integer before, rather than use `toString()` into the loop ?

Reply:
Yep for the number casing !

Reply:
About your proposal on `parentId` I'm afraid it will only work if the filtered ID is for a single category.
If the ID is from a category group, we don't want the actual item but it's children right ?

Reply:
Right.